### ISSUE TITLE
Can't join the active group 'try'

### ISSUE THREAD

**Issue by Ashe** [braidchat/braid#194]
I want to get in touch and help out but when I log in and click on the `try` active group, nothing happens. I actually really want this to succeed as I am ***sick*** of discord, nice to see something new on the rise. Really impressed by the trailer too, and I really hope I can learn while contributing. I hope you'll have me!

**Comment by Ashe** [comment]
Also: I'm sorry that this is posted here even though it's not an actual issue with the programming side of things, but its hard to communicate with you all when you're sealed away on your own platform that I seemingly cannot join :(

**Comment by jamesnvc** [comment]
Hi Ashe, no worries, thanks for reporting an issue!

Some questions:
 - What browser & OS are you using? 
 - When do you see the error? Can you post a screenshot of what's happening whet you get the error

**Comment by Ashe** [comment]
Thanks for getting back to me so quickly!
I am using Arch Linux and Firefox Quantum 68.0 (64-bit).

[Here's a temporary webm I recorded of the error.](https://a.uguu.se/VR7oaSzwCs5Z_BraidError)

I also tried on Google chrome and has the same issue sadly.

**Comment by jamesnvc** [comment]
Oh, very strange...I'm having trouble reproducing that right now, but I'll look into that.

Can you go directly to https://braid.chat/try?

**Comment by Ashe** [comment]
Unfortunately not. Unless this is how it's supposed to look:

![image](https://user-images.githubusercontent.com/8940077/61157852-341d8680-a4ef-11e9-83d0-e7c2091c1a2e.png)

I did try that but I didn't know if that was the url address to the group.

**Comment by jamesnvc** [comment]
Yup, that URL looks right...I saw you were able to create a group, though?

**Comment by jamesnvc** [comment]
Can you try opening a new incognito window and going to braid.chat/try? I'm wondering if the app state has gotten in to some sort of strange state for you...

**Comment by Ashe** [comment]
Yeah, you can try [braid.chat/ashe](https://braid.chat/ashe)? But when I access that link I get the blank screen again, just like in the screenshot above. I'll be in my group if you wanted to try and jump in? It should be public and at least we'd see if the basics work.

**Comment by Ashe** [comment]
[Here's an invite link I generated for you?](https://braid.chat/invite?group-id=5d28de26-59c5-4de2-b75a-f0b630d3e8b6&nonce=uHxNOolfn7GRbm-NtpzY_ZjaAoE&expiry=1563051746742&mac=b331f92016edb52b46a311513ecdf1a4cc8645339a3b0f6784808facb25d737a) Maybe it'll help you get in if the standard url doesn't work.

**Comment by Ashe** [comment]
You were right! I saw the group perfectly fine in incognito mode. So I wonder what's going on... I'll try logging out of my account and trying your group again

**Comment by jamesnvc** [comment]
Yeah, when I try to go to your "Ashe's group" then I get that error. I've seen things about that before, there's probably some weird thing with how a message was created in there. I'll poke around in the db & see if I can figure it out.

**Comment by Ashe** [comment]
Okay, I managed to join `try`, and I tagged `jamesnvc` into my conversation. Does it work?
